WBooks/Tabs/Library/LibraryController.swift

```diff
@@ -38,19 +38,19 @@ class MainMenuController: BaseViewController {
     
     private func configureNavigationBar() {
```

Comment:
You can add "// MARK: - Private" before private methods to maintain an order, it will be easier to find when the class has a thousand lines xD